test/TEST-30-ONCLOCKCHANGE/test.sh

```diff
@@ -30,6 +30,7 @@ test_setup() {
         ln -fs /dev/null $initdir/etc/systemd/system/systemd-machined.service
 
         # extend the watchdog
+        mkdir $initdir/etc/systemd/system/systemd-timedated.service.d
```

Comment:
It passed before, which suggests that the directory already existed. So maybe do `mkdir -p` to make it work irrespectively?

Reply:
Yeah. Updated.